scoreboard/controllers.py

```diff
@@ -82,6 +82,34 @@ def register_user(email, nick, password, team_id=None,
                     nick, email, flask.request.remote_addr)
     return user
 
+@utils.require_not_started
+def change_user_team(uid, team_tid, code):
```

Comment:
There's a slight edge case if people do starting and stopping and the scoreboard. Seems unlikely, but worthwhile adding a check that the original team has no answers at the moment.